src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,234 @@
+											{{ loan.sector?.name || '-' }}
+										</div>
+										<div class="tw-flex tw-items-center tw-text-secondary">
+											<div class="tw-w-2 tw-h-2 tw-mr-1">
+												<kv-flag
+													v-if="loan.geocode?.country?.isoCode"
+													:country="loan.geocode?.country?.isoCode"
+												/>
+											</div>
+											{{ loan.geocode?.country?.name || '-' }}
+										</div>
+										<div class="tw-text-secondary" v-if="loan.partnerName">
+											<a
+												:href="getPartnerUrl(loan.partnerId)"
+												target="_blank"
+												rel="noopener noreferrer"
+											>
+												{{ loan.partnerName }}
+											</a>
+										</div>
+									</div>
+								</div>
+							</td>
+							<td class="tw-px-2">
+								<div class="tw-text-secondary">
+									{{ getPrintableStatus(loan.status) }}
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountPurchasedByLender }}
+									</div>
+									<div class="tw-mb-1 tw-text-secondary">
+										{{ formatDate(loan.terms.disbursalDate) || '(Endpoint TBD)' }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div>
+										${{ loan.terms.loanAmount }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									{{ loan.lenderRepaymentTerm || '-' }} mos
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountRepaidToLender }} repaid to you
+									</div>
+								</div>
+							</td>
+							<td class="tw-max-w-[150px] tw-whitespace-normal tw-break-words tw-px-2">
+								<div class="tw-items-center">
+									<template v-if="loan.teams?.values?.[0]">
+										<img
+											v-if="loan.teams.values[0].image?.url"
+											:src="loan.teams.values[0].image.url"
+											:alt="`${loan.teams.values[0].name} team image`"
+											class="tw-w-5 tw-h-5"
+										>
+										<span>{{ loan.teams.values[0].name }}</span>
+									</template>
+									<span v-else>-</span>
+								</div>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div class="scroll-gradient"></div>
+		</div>
+	</div>
+</template>
+
+<script>
+import { KvFlag, KvLoadingPlaceholder } from '@kiva/kv-components';
+
+export default {
+	name: 'LoanList',
+	inject: ['cookieStore'],
+	props: {
+		loans: {
+			type: Array,
+			default: () => []
+		},
+		loading: {
+			type: Boolean,
+			default: true
+		}
+	},
+	components: {
+		KvFlag,
+		KvLoadingPlaceholder
+	},
+	methods: {
+		formatDate(date) {
+			if (!date) return '';
+			return new Date(date).toLocaleDateString('en-US', {
+				month: 'short',
+				day: 'numeric',
+				year: 'numeric'
+			});
+		},
+		getPrintableStatus(rawStatus) {
```

Comment:
I think we can bring in these definitions from: src/api/fixtures/LoanStatusEnum.js